[JS] Add a Q/A section for CAP LLM Plugin (#150)
SAP/ai-sdk: docs-js/faq.mdx

---
id: frequently-asked-questions
title: Frequently Asked Questions
hide_title: false
hide_table_of_contents: false
sidebar_label: FAQ

> description: You asked! We answered! We collected here the most frequent question about SAP Cloud SDK for JavaScript.

Reviewer: Here I would add "for AI" to the products title

> description: You asked! We answered! We collected here the most frequent question about SAP Cloud SDK for JavaScript.

Author: ```suggestion
description: You asked! We answered! We collected here the most frequent question about SAP Cloud SDK for AI (JavaScript).
```

keywords:
  - sap
  - cloud
  - sdk
  - ai
---

### Which library should I use for AI integration in the SAP ecosystem? I see an npm package `CAP LLM Plugin`.

> Use the **SAP Cloud SDK for AI**, the only official Software Development Kit (SDK) for **SAP AI Core**, **SAP Generative AI Hub**, and **Orchestration Service**.

Reviewer: I would phrase it a little less confrontative, therefore remove the "only" and maybe phrase it more like:

"Use the **SAP Cloud SDK for AI**, as it is the official Software Development ..."

> Use the **SAP Cloud SDK for AI**, the only official Software Development Kit (SDK) for **SAP AI Core**, **SAP Generative AI Hub**, and **Orchestration Service**.

Author: ```suggestion
Use the **SAP Cloud SDK for AI**, the official Software Development Kit (SDK) for **SAP AI Core**, **SAP Generative AI Hub**, and **Orchestration Service**.
```

The SAP Cloud SDK for AI is fully aligned with the SAP AI Core roadmap, ensuring you always have access to the latest features and the newest LLMs.

> The `CAP LLM Plugin` includes the following disclaimer:

Reviewer: Remove the whole disclaimer, as for the above mentioned reason.

> 
> The `CAP LLM Plugin` includes the following disclaimer:
> 
> > This project is provided "as-is" with no expectation for major changes or support.
> 
> As a result, new models (e.g., `gpt-4.1`, `gemini-2.0`, `anthropic--claude-3.7`, `mistralai`, and `deepseek`) and upcoming features of SAP AI Core will not be supported.
> 

Author: ```suggestion

```

> Only use the plugin for special use cases or if you are an existing user.

Reviewer: Only use the `CAP LLM Plugin` for special use cases or if you are an existing user, as the plugin cannot provide support for other use-cases and will stay with a limited scope and feature set.

> This recommendation is fully aligned with the CAP LLM Plugin team.

Reviewer: This should be be obvious, so let's remove it.

> This recommendation is fully aligned with the CAP LLM Plugin team.

Reviewer: Maybe add just a quick not that there are also Cloud SDK's for other languages as official tooling available?

> Only use the plugin for special use cases or if you are an existing user.
> This recommendation is fully aligned with the CAP LLM Plugin team.

Author: ```suggestion
Only use the `CAP LLM Plugin` for special use cases or if you are an existing user, as the plugin cannot provide support for other use-cases and will stay with a limited scope and feature set.
```
